samples/boards/st/uart/single_wire/README.rst

```diff
@@ -7,14 +7,18 @@
 Overview
 ********
 
-A simple application demonstrating how to use the single wire / half-duplex UART
-functionality of STM32. Without adaptions this example runs on STM32F3 discovery
-board. You need to establish a physical connection between pins PA2 (USART2_TX) and
-PC10 (UART4_TX).
-
+A simple application demonstrating how to use the single-wire/half-duplex
+UART functionality of STM32 devices. The example runs on various STM32
+boards with minimal adaptations.
 Add a ``single_wire_uart_loopback`` fixture to your board in the hardware map to allow
 twister to verify this sample's output automatically.
 
+Hardware Setup
+********
```

Comment:
```suggestion
Hardware Setup
**************
```

Required to solve https://github.com/zephyrproject-rtos/zephyr/actions/runs/13160412192/job/36727517154?pr=82946#step:8:66